apps/api-extractor/src/schemas/api-extractor.schema.json

```diff
@@ -31,6 +31,13 @@
       }
     },
 
+    "enumMemberOrder": {
+      "description":  "Specifies how API Extractor sorts the members of an enum when generating the .d.ts rollup and .api.json doc model. \n 'by-name': sort the items according to the enum member name \n 'preserve': keep the original order that items appear in the source code",
```

Comment:
This description needs to be updated as it doesn't affect the `.d.ts` rollup, right?